docs/nixos-with-flakes/other-useful-tips.md

```diff
@@ -9,8 +9,8 @@ However, by default, NixOS places the configuration in `/etc/nixos`, which requi
 For example, you can place your flake in `~/nixos-config` and create a symbolic link in `/etc/nixos` as follows:
 
 ```shell
-sudo mv /etc/nixos /etc/nixos.bak  # Backup the original configuration
-sudo ln -s ~/nixos-config/ /etc/nixos
```

Comment:
And the purpose here is to create a symlink `/etx/nixos`, which links to `~/nixos-config/`, no changes need to be made here too.